packages/ai-jsx/readme.md

```diff
@@ -27,7 +29,7 @@ For more details on how AI.JSX works with React in general, see our [AI+UI guide
 
 ## Examples
 
-You can play with live demos on our [live demo app](https://ai-jsx-nextjs-demo.vercel.app/) ([source](../nextjs-demo/)).
+You can play with live demos on our [live demo app](https://ai-jsx-nextjs-demo.vercel.app/) (source is available [here](./packages/nextjs-demo/)).
```

Comment:
I for one really dislike the "link the word `here`" pattern. It's worse for screenreaders, among other things. But I won't hold up the PR on it. :smile:

Reply:
happy to change this. what would you suggest otherwise?

Reply:
will update this